package.json

```diff
@@ -24,20 +15,64 @@
     "internet of things",
     "serverless.com"
   ],
+  "eslintConfig": {
+    "extends": "@serverless/eslint-config/node",
+    "root": true
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint"
+    ],
+    "*.{css,html,js,json,md,yaml,yml}": [
+      "prettier -c"
+    ]
+  },
+  "standard-version": {
+    "skip": {
+      "commit": true,
+      "tag": true
+    }
+  },
   "dependencies": {
     "async": "^2.6.3",
-    "bluebird": "^3.5.5",
+    "bluebird": "^3.7.2",
     "chalk": "^2.4.2",
     "fs-extra": "^8.1.0",
-    "googleapis": "^41.0.1",
+    "googleapis": "^39.2.0",
```

Comment:
Is this breaking since it's a downgrade?

Reply:
I don't think so. All tests pass.

Also upgrade from v32 to v41 was made without any extra changes to code: https://github.com/serverless/serverless-google-cloudfunctions/pull/176/files

And that was actually breaking (as it dropped Node.js v6 support, CI didn't pick that as its setup was broken - just lint was run, no tests).

So this is an attempt to bring back support for Node.js v6 in v2 scope, which we will drop with v3

Reply:
Makes sense. Thanks for the clarification :+1: